lib/Dancer2/Manual.pod

```diff
@@ -213,6 +213,13 @@ coderef to execute, which returns the response.
 The above route specifies that, for GET requests to C</hello/...>, the code
 block provided should be executed.
 
+
+You can also provide routes with a name:
+
+    get 'hi_to' => '/hello/:name' => sub {...};
+
+See C<uri_for_route> on how this can be used.
```

Comment:
Suggestion:
```suggestion
See L<Dancer2::Manual::Keywords/uri_for_route> on how this can be used.
```
(and likewise in other pod outside D2::M::K)

Reply:
Good call! Fixed.